### PR TITLE
typescript package to run quorum policy transactions

### PR DESCRIPTION
**Pull request by dariorussi**
Define the TypeScript code to run all transactions related to quorum upgrade policy

### CODE REVIEW

File: upgrade_policy/quorum_upgrade/scripts/ts/.DS_Store
Comment (damirka) [inline upgrade_policy/quorum_upgrade/scripts/ts/.DS_Store:1]: Do you want this buddy in?

Comment (dariorussi) [reply upgrade_policy/quorum_upgrade/scripts/ts/.DS_Store:1]: no I thought I removed it and it's still there.
Let me kill it and thanks!

**Review by damirka — APPROVED** [review]
Looks good to me. I wonder what @manolisliolios has to say about it as he's more versed in TypeScript and tooling.

**Comment by dariorussi** [comment]
> Looks good to me. I wonder what @manolisliolios has to say about it as he's more versed in TypeScript and tooling.

though that is mostly his code ;)

File: upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+// Voters. Add all addresses that will be part of te quorum policy.
+const VOTER_1 = '';
+const VOTER_2 = '';
+const VOTER_3 = '';
+// The package id of the `quorum_upgrade_policy` package (published on testnet)
+const QUORUM_UPGRADE_PACKAGE_ID = `0x9722f37d05ba8dd7295e0828edf00849aea17411e76abc3f544ad22069cae733`; 
+// The upgrade cap of the quorum upgrade policy (resulting from `quorum_upgrade_policy::new`)
+const QUORUM_UPGRADE_CAP_ID = ``;
+// path to the package to publich or upgrade
```
Comment (manolisliolios) [inline upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts:17]:
```suggestion
// path to the package to publish or upgrade
```

File: upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
+
+import {TransactionBlock} from "@mysten/sui.js/transactions";
+import { getUpgradeDigest, getActiveAddress, prepareAddressVecSet, signAndExecute } from "./utils";
+
+// =================================================================
+// Constant to update when running the different transactions
+// =================================================================
+
+// Voters. Add all addresses that will be part of te quorum policy.
```
Comment (manolisliolios) [inline upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts:9]:
```suggestion
// Voters. Add all addresses that will be part of the quorum policy.
```

File: upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
+
+import {TransactionBlock} from "@mysten/sui.js/transactions";
+import { getUpgradeDigest, getActiveAddress, prepareAddressVecSet, signAndExecute } from "./utils";
+
+// =================================================================
+// Constant to update when running the different transactions
```
Comment (manolisliolios) [inline upgrade_policy/quorum_upgrade/scripts/ts/src/transactions.ts:6]:
```suggestion
// Constants to update when running the different transactions
```

**Review by manolisliolios — APPROVED** [review]
Thanks, lgtm

**Comment by dariorussi** [comment]
feedback

**Pull request merged**